cmd/openshift-install/create.go

```diff
@@ -550,9 +555,11 @@ func logComplete(directory, consoleURL string) error {
 		return err
 	}
 	logrus.Info("Install complete!")
-	logrus.Infof("To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", kubeconfig)
-	logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
-	logrus.Infof("Login to the console with user: %q, and password: %q", "kubeadmin", pw)
+	if !createClusterOpts.suppressLoginInfo {
+		logrus.Infof("To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", kubeconfig)
+		logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
```

Comment:
Do these lines need to be suppressed? Or is it only the line containing the username and password for the console?

Reply:
My preference is to hide as well such url

Reply:
Can you elaborate? My reasoning for not wanting to hide the console URL is that the URL is useful to the end user and not available elsewhere in the installation directory. On that note, I would prefer that the username/password line be replaced with a line directing the user to the location of the file containing the password.

Reply:
Sure, especially in a CI job context that piece of info can be extracted programmatically in some other way (ie through a `oc -n openshift-console get routes console -o=jsonpath='{.spec.host}'`), so no need to print out when no human interaction is required, minimizing thus the info provided.
The idea of the flag was to preserve the current existing default behavior.